scim/resource_user.go

```diff
@@ -93,7 +100,28 @@ func ResourceUser() *schema.Resource {
 			return NewUsersAPI(ctx, c).Update(d.Id(), u)
 		},
 		Delete: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
-			return NewUsersAPI(ctx, c).Delete(d.Id())
+			user := NewUsersAPI(ctx, c)
+			userName := d.Get("user_name").(string)
+			err := user.Delete(d.Id())
+			if err != nil {
+				return err
+			}
+			if c.Config.IsAccountClient() && c.Config.AccountID != "" {
+				return nil
+			}
+			if d.Get("force_delete_repos").(bool) {
+				err = NewUsersAPI(ctx, c).DeletePath(fmt.Sprintf("/Repos/%v", userName))
```

Comment:
initialize workpsace api, don't add new method in the users API. it doesn't exist in the SDK.